features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/WSDL/ApiCreateWSDL.js

```diff
@@ -17,16 +17,18 @@
  */
 import React, {Component} from 'react';
 /* MUI Imports */
-import {Grid, Paper, Button, Divider, Typography} from 'material-ui'
-import Arrow from 'material-ui-icons/ArrowForward'
 import {withStyles} from 'material-ui'
-import {FormControl, Input, InputLabel, FormHelperText} from 'material-ui'
-import CreateAPI from '../Endpoint/ApiCreateEndpoint'
+import {Grid, Button, Typography} from 'material-ui'
```

Comment:
do not import like this since we are doing multiple calls

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/4999/commits/725ff244f606c3eb6ad6ae7b88f9ff91b820b50e